core/src/main/java/io/questdb/std/str/Utf8s.java

```diff
@@ -714,6 +714,23 @@ public static boolean utf8ToUtf16Z(long lo, CharSinkBase<?> sink) {
         return true;
     }
 
+    /**
+     * Copies UTF8 zero-terminated string into UTF8 sink excluding zero byte.
```

Comment:
nit: we call it "NULL-terminated" in other places
```suggestion
     * Copies UTF8 NULL-terminated string into UTF8 sink excluding zero byte.
```

Reply:
done in https://github.com/questdb/questdb/pull/4137/commits/4f372f37e5cf4d5b7f9e8374383a72b79d8eee10